spec/goss/goss.yaml

```diff
@@ -0,0 +1,5 @@
+file:
+  /app/startup.sh:
+    exists: true
+    filetype: file
+    contains: []
```

Comment:
Maybe there could be some more tests. Look for an example at: https://github.com/cloudogu/cas/blob/develop/spec/goss/goss.yaml